docs/table-partitioning-bucketing-sort-order.md

```diff
@@ -0,0 +1,288 @@
+      "score"
+    ]
+  }
+]
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+new Transform[] {
+    // Partition by score
+    Transforms.identity("score")
+    }
+```
+
+</TabItem>
+</Tabs>
+
+
+## Bucketed table
+
+- Strategy. It defines how Gravitino will distribute table data across partitions.
+
+| Bucket strategy | Description                                                                                                                   | JSON     | Java             |
+|-----------------|-------------------------------------------------------------------------------------------------------------------------------|----------|------------------|
+| hash            | Bucket table using hash. Gravitino will distribute table data into buckets based on the hash value of the key.                | `hash`   | `Strategy.HASH`  |
+| range           | Bucket table using range. Gravitino will distribute table data into buckets based on a specified range or interval of values. | `range`  | `Strategy.RANGE` |
+| even            | Bucket table using even.  Gravitino will distribute table data, ensuring an equal distribution of data.                       | `even`   | `Strategy.EVEN`  |
+
+- Number. It defines how many buckets you use to bucket the table.
+- Function arguments. It defines the arguments of the strategy above, Gravitino supports the following three kinds of arguments, for more, you can refer to Java class [FunctionArg](https://github.com/datastrato/gravitino/blob/main/common/src/main/java/com/datastrato/gravitino/dto/rel/expressions/FunctionArg.java) and [DistributionDTO](https://github.com/datastrato/gravitino/blob/main/common/src/main/java/com/datastrato/gravitino/dto/rel/DistributionDTO.java) to use more complex function arguments.
+
+| Expression type | JSON example                                                   | Java example                                                                              | Equivalent SQL semantics | Description                       | 
+|-----------------|----------------------------------------------------------------|-------------------------------------------------------------------------------------------|--------------------------|-----------------------------------|
+| field           | `{"type":"field","fieldName":["score"]}`                       | `FieldReferenceDTO.of("score")`                                                           | `score`                  | The field reference value `score` |
+| function        | `{"type":"function","functionName":"hour","fieldName":["dt"]}` | `new FuncExpressionDTO.Builder().withFunctionName("hour").withFunctionArgs("dt").build()` | `hour(dt)`               | The function value `hour(dt)`     |
+| constant        | `{"type":"literal","value":10, "dataType": "integer"}`         | `new LiteralDTO.Builder().withValue("10").withDataType(Types.IntegerType.get()).build()`  | `10`                     | The integer literal `10`          |
```

Comment:
The content here has no any introduction about bucketing, directly introducing Strategy, Number... is very hard for user to understand what is it. You should have a basic introduction about the bucketing, and the required fields of bucketing.

Also not only for bucketing, but also for partitioning and sort ordering.

Reply:
This part is in the `manage-metadata-using-gravitino.md` and I have updated it.

Reply:
I mean you'd better have a paragraph introducing all the required fields to combine partitioning, bucketing and sort ordering. For a user who doesn't have any background of these things, directly introducing `Strategy`, `Number` and others seems not so easy to understand.

Reply:
added.